syncstorage/src/web/middleware/sentry.rs

```diff
@@ -186,11 +200,12 @@ pub fn event_from_error(err: &ApiError) -> Event<'static> {
 /// Custom `exception_from_error` support function for `ApiError`
 ///
 /// Based moreso on sentry_failure's `exception_from_single_fail`.
-fn exception_from_error_with_backtrace(err: &ApiError) -> sentry::protocol::Exception {
+fn exception_from_error_with_backtrace<E>(err: &E) -> sentry::protocol::Exception
+where
+    E: ReportableError + StdError + ?Sized,
```

Comment:
Looks like this `?Sized` isn't needed
```suggestion
    E: ReportableError + StdError,
```